modules/core/src/persistence.cpp

```diff
@@ -4025,7 +4025,14 @@ static void
 icvJSONWriteReal( CvFileStorage* fs, const char* key, double value )
 {
     char buf[128];
-    icvJSONWrite( fs, key, icvDoubleToString( buf, value ));
+    int len = (int)strlen( icvDoubleToString( buf, value ) );
```

Comment:
`size_t` is the actual return type of `strlen`, not an `int`. So if anything is forced here, it's the type cast to an `int`. In my opinion, bad style should not be duplicated. Someone may copy the code in the future again which may cause problems.

Nevertheless, feel free to merge the patch as is.